FirstProject/Program.cs

```diff
@@ -59,18 +60,24 @@
             case 2:
                 Console.WriteLine("Obliczenie silni z liczby");
                 value1 = Convert.ToInt32(Console.ReadLine());
-                Console.WriteLine(factorial.Factorial(value1));
+                Console.WriteLine(factorial.GetFactorial(value1));
                 break;
             case 3:
                 Console.WriteLine("Kalkulator potęg");
                 Console.WriteLine("Podaj podstawę potęgi");
                 value1 = Convert.ToInt32(Console.ReadLine());
                 Console.WriteLine("Poda wykładnik potęgi");
                 value2 = Convert.ToInt32(Console.ReadLine());
-                Console.WriteLine(power.Power(value1, value2));
+                Console.WriteLine(power.GetPower(value1, value2));
+                break;
+            case 4:
+                Console.WriteLine("Czy liczba jest liczba doskonala?");
```

Comment:
Chyba lepiej by było coś ala "Podaj liczbę" bo to co napisałeś nic mi jako użytkownikowi nie mówi co mam zrobić